staging/src/k8s.io/client-go/kubernetes/typed/admissionregistration/v1/generated_expansion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 Authors of Arktos.
+Copyright The Kubernetes Authors.
```

Comment:
make update put this up for likely a bug in the boiler update script. will revert this file.